Sitefinity-VSIX/vNextPackage.cs

```diff
@@ -48,6 +48,14 @@ protected async override System.Threading.Tasks.Task InitializeAsync(Cancellatio
                             process.StartInfo.Arguments = args;
                             process.Start();
                             process.WaitForExit();
+
+                            if (process.ExitCode != 0)
+                            {
+                                var message = Constants.GeneralErrorMessage;
```

Comment:
I do not like the fact that we always show the **GeneralErrorMessage** when the cause of the error can be something different that the config file generation.